pspamm/codegen/architectures/hsw/inlineprinter.py

```diff
@@ -64,7 +64,15 @@ def visitMul(self, stmt: MulStmt):
     def visitBcst(self, stmt: BcstStmt):
         b = stmt.bcast_src.ugly
         a = stmt.dest.ugly
-        s = "vbroadcasts{} {}, {}".format(self.precision, b,a)
+        # check if we broadcast a general register
+        if b[2] == 'r':
+            # determine if we broadcast alpha or beta
+            src = "%3" if stmt.bcast_src.ugly == "%%rbx" else "%4"
```

Comment:
Can't we do a move `%3` and `%4` into rbx and rcx in advance maybe (and then just forward them)?